### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by raphaellueckl**

### CODE REVIEW

File: src/components/20-molecules/image-upload/README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+- To upload a file, you either drag an drop your file, or click the input-file component.
+- The image-upload has two states. The start view and the file overview. The start view contains a text which asks the user to drag and drop a file, an icon consisting of four subjects and the input-file-component with an icon. The file overview shows all files ready to upload. Each file has a caption and when hovering over it displays the bin-icon to delete the file. Is the file to big or is of the wrong type, a red icon will be displayed and the caption will be replaced through the red error message. Nevertheless, when hovering over, it shows the bin-icon.
```
Comment (NoahLiechti) [inline src/components/20-molecules/image-upload/README.md]: "file-upload"

File: src/components/20-molecules/image-upload/README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
 # Image upload
```
Comment (NoahLiechti) [inline src/components/20-molecules/image-upload/README.md]: "file-upload"

File: src/components/20-molecules/image-upload/README.md [hunk 3]
```diff
@@ -69,37 +69,37 @@ Import the image-upload-defining script and use a image-upload like this:
...
-| `addStatusText="Add more"`                 | Specify the caption of dashed box                                    |
-| `infoText="drag and drop your files here"` | Specify the text on the start view                                   |
-| `orText="or"`                              | Specify the text on the start view box                               |
+| Attribute                                  | Details                                                               |
+| ------------------------------------------ | --------------------------------------------------------------------- |
+| `inputFileText="Upload file"`              | Text in the input-file component                                      |
+| `maxSizeOfSingleFileKB="10"`               | Maximal size of a single file in KB                                   |
+| `maxSizeOfAllFilesKB="15"`                 | Maximal size of a all files together in KB                            |
+| `maxNumberOfFiles="10"`                    | Maximal number of files                                               |
+| `showImageOverview="false"`                | Switch between normal view and image overview                         |
```
Comment (NoahLiechti) [inline src/components/20-molecules/image-upload/README.md]: would it make sense to call it file overview? Bc we are renaming the image-upload anyway. And technically it does show images and files

File: src/components/20-molecules/image-upload/README.md [hunk 4]
```diff
@@ -69,37 +69,37 @@ Import the image-upload-defining script and use a image-upload like this:
...
 
-The attribute `maxSizeOfAllFilesKB` specifies the maximal size all compressed files together can have. Every File that is over the limit gets displayed as a wrong file.
+The attribute `maxSizeOfAllFilesKB` specifies the maximal size all compressed files together can have. Every file that is over the limit gets displayed as a wrong file.
 
 ### maxNumberOfFiles
 
-The attribute `maxNumberOfFiles` specifies the maximal number of files. Every File that is over the limit will not be displayed.
+The attribute `maxNumberOfFiles` specifies the maximum number of files. Every File that is over the limit will not be displayed.
 
 ### showImageOverview
```
Comment (NoahLiechti) [inline src/components/20-molecules/image-upload/README.md]: same here "showFileOverview"

File: src/components/20-molecules/image-upload/README.md (hunk 3 above)
Comment (raphaellueckl) [reply src/components/20-molecules/image-upload/README.md]: Yes rename that stuff. But that has to go with the change in the code, not that code and readme diverges.

**Pull request merged**